microcosm_flask/swagger/schema.py

```diff
@@ -38,6 +38,16 @@
 }
 
 
+SWAGGER_TYPE = "__swagger_type__"
+SWAGGER_FORMAT = "__swagger_format__"
+
+
+def swagger_field(field, swagger_type="string", swagger_format=None):
```

Comment:
I added a wrapper function for a marshmallow field so we can more easily attach the customization attributes.